apps/docs/src/packages/storybook-utils.md

```diff
@@ -111,3 +113,62 @@ type Story = StoryObj<typeof TestInput>;
 ```
 
 :::
+
+### createTheme
+
+Creates a custom theme for Storybook that uses Onyx colors.
+See the [Storybook Theming docs](https://storybook.js.org/docs/configure/theming) for further information.
+
+::: tip
+If you are using [`createPreview()`](#createpreview), the custom light and dark theme will already be set up for you.
```

Comment:
![image](https://github.com/SchwarzIT/onyx/assets/151019977/51650ceb-bea6-4819-864f-d301b4bbc873)

is the "tip" background color also given by our design tokens? 
imho the `createPreview()` is veery hard to read with the green on purple background :(

fyi @jannick-ux

Reply:
dark mode is fine:
![image](https://github.com/SchwarzIT/onyx/assets/151019977/a83ab739-b905-483e-a3a8-d0fe7c8659c4)

Reply:
This is about VitePress, not Storybook right? This should be fixed with #131

Reply:
ah wasn't up to date with main yet, I see :)
![image](https://github.com/SchwarzIT/onyx/assets/151019977/83c40496-6b17-48d4-a34b-a34f7af534ff)
still not the best contrast as the transparent green backgrounds add up, but better than before.

Reply:
I updated the color to improve contrast :)